regression/tests/enterprise/ent_test_base.py

```diff
@@ -201,3 +229,28 @@ def register_and_go_to_course_enrollment_page(self):
         self.register_ent_edx_user()
         # Verify that user is on course enrollment page
         self.ent_course_enrollment.wait_for_page()
+
+    def payment_using_cyber_source(self):
+        """
+        Make payment for course by providing Billing Info and Payment details
+        in respected areas.
+        """
+        self.cyber_source_page.set_card_holder_info(CARD_HOLDER_INFO)
+        self.cyber_source_page.set_billing_info(BILLING_INFO)
+        self.cyber_source_page.click_payment_button()
+        self.receipt_page.wait_for_page()
```

Comment:
don't we have this function already write somewhere?

Reply:
This function is outside of enterprise package, so we need to add this in our enterprise base class in order to use.